com.unity.perception/Runtime/GroundTruth/SemanticSegmentationCrossPipelinePass.cs

```diff
@@ -33,11 +33,20 @@ public override void Setup()
         {
             base.Setup();
             m_ClassLabelingShader = Shader.Find(k_ShaderName);
+
             var shaderVariantCollection = new ShaderVariantCollection();
-            shaderVariantCollection.Add(new ShaderVariantCollection.ShaderVariant(m_ClassLabelingShader, PassType.ScriptableRenderPipeline));
-            shaderVariantCollection.WarmUp();
+
+            if (shaderVariantCollection != null)
+            {
+                shaderVariantCollection.Add(new ShaderVariantCollection.ShaderVariant(m_ClassLabelingShader, PassType.ScriptableRenderPipeline));
+            }
 
             m_OverrideMaterial = new Material(m_ClassLabelingShader);
+
+            if (shaderVariantCollection != null)
```

Comment:
Will it ever be null?

Reply:
I just have trouble seeing stuff like this in engine code - as in, not much error handling.  I know the underlying engine is generally something we can depend on, but I'm used to always adding debug null checks which get removed in release/production

Reply:
In general we have the benefit of using very established features that generally follow their public contracts :)

Reply:
Makes sense - this was updated in a prior change, but github has not updated it here.